doc/api_queries.rst

```diff
@@ -110,6 +114,11 @@
 
     Evaluate the query. Returns a list of matching :class:`dnf.package.Package` instances.
 
+  .. method:: unneeded(sack, yumdb, debug_solver=False)
```

Comment:
don't think that it's good idea to expose functions with yumdb argument

about debug_solver also. For debug solver we should have another function/parameter which will enable debugging for solver.